### PR TITLE
updated pathfinding limit to 20

### PR DESCRIPTION
**Pull request by nikhilsaraf**
closes https://github.com/stellar/horizon/issues/409

### CODE REVIEW

**Comment by nikhilsaraf** [comment]
travis build failed because of prod network slowness. triggered rebuild by touching file.

**Comment by nikhilsaraf** [comment]
is this good to merge? @nullstyle or @jedmccaleb

File: services/horizon/internal/simplepath/search.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func (s *search) hasMore() bool {
 		return false
 	}
 
-	if len(s.Results) > 4 {
+	if len(s.Results) >= 20 {
```
Comment (nullstyle) [inline services/horizon/internal/simplepath/search.go]: Let's not just up this magic number.  Instead, let's make it configurable.

Comment (nikhilsaraf) [reply services/horizon/internal/simplepath/search.go]: extracted to a constant for now as we discussed.

**Pull request merged**